test/functional/tests/conftest.py

```diff
@@ -184,6 +187,10 @@ def base_prepare(item):
                         Udev.settle()
 
         RamDisk.remove_all()
+
+        if check_if_directory_exists("/tmp/open_cas_test_data"):
```

Comment:
Please make this path a const. It is used in few places. Then it could be used also in tests